frontend/test/metabase/scenarios/visualizations/table.cy.spec.js

```diff
@@ -153,10 +154,10 @@ describe("scenarios > visualizations > table", () => {
     summarize();
 
     cy.findAllByTestId("dimension-list-item-name")
-      .first()
+      .contains(ccName)
       .click();
 
-    cy.icon("table2").click();
```

Comment:
Just for history. What exactly is/was icon(table2)?

Reply:
It was clicking on the table view.

<img width="1060" alt="Screen Shot 2022-04-13 at 07 14 21" src="https://user-images.githubusercontent.com/17742979/163156735-2df7a36f-f980-4e7a-a458-edfaa1bdc8ff.png">

Reply:
That's RawData toggle.

Reply:
Yeah, it doesn't do anything now because we already render results as a table.